apps/codelab/src/app/modules/admin/services/can-activate-child.guard.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { CanActivateChild } from '@angular/router';
-import { AccessService, Permissions } from './access.service';
+import { AccessService, Permissions } from '../../../../../../../libs/firebase-login/src/lib/access.service';
```

Comment:
I think you can just use here
import { AccessService } from '@codelab/firebase-login';